app/controllers/stories/articles_search_controller.rb

```diff
@@ -0,0 +1,22 @@
+module Stories
```

Comment:
I'm not fully convinced by the naming here. This is in no way your fault but it was always confusing that the model was called `Article` while the corresponding controller was named `StoriesController`. On top of that, we're planning to rename the model to `Post` eventually. 😅 I'd be tempted to have this in `Articles::SearchController` instead because it's articles we're searching for.

Reply:
My first though was to move it to a new `controller/articles` folder, however there are three controllers dealing with articles in the `controller/stories` folder. I assume "stories" to be a set of activities dealing with different types of content (posts, podcasts, pages, organizations etc). There's also `find_featured_story` in `app/services/articles/feeds/large_forem_experimental.rb` that deals with featuring articles on users home feed according to user preferences and activity. I foundit rather difficult to interpret what the term "stories" stand for. As you say, creating controllers in modules that correspond to the model name for every such activity in stories_controller seems correct. However, I would like to have some more clarity on what stories is :sweat_smile:

Reply:
It's a problem we had in several areas of our codebase, names didn't agree in several places. For example, what is called `Listings` in the front end was backed by a model called `ClassifiedListing` which I eventually ended up renaming. "Stories" is not really a word we use in internal discussions, so it's not really a domain concept. Hence the pending rename to `Post`. In the meantime, I think `Articles::Search` would make more sense but I am aware that there are already several article-related controllers in the `Stories` namespace. I guess we might as well just leave it here.